tensorflow/python/ops/array_ops.py

```diff
@@ -5871,3 +5871,55 @@ def repeat(input, repeats, axis=None, name=None):  # pylint: disable=redefined-b
     input = reshape(input, [-1])
     axis = 0
   return repeat_with_axis(input, repeats, axis, name)
+
+@tf_export(v1=["mask_fill"])
+@dispatch.add_dispatch_support
+def mask_fill(tensor, mask, name="mask_fill", value=0):
+  """Apply boolean mask fill to tensor.
+  
+  In general, an array `tesnsor` is updated to the `value` where 
+  corresponding index have `True` or `1` entry in `mask` array.
+
+  Examples:
+
+  ```python
+  >>>tensor = [0, 1, 2, 3]
+  >>>mask = np.array([True, False, True, False])
+  >>>value = 5
+  >>>tf.mask_fill(tensor, mask, value)  # [5, 1, 5, 3]
+  ```
```

Comment:
```suggestion
  >>> tensor = tf.constant([0, 1, 2, 3])
  >>> mask = np.array([True, False, True, False])
  >>> tf.mask_fill(tensor, mask, 5)
  [5, 1, 5, 3]
```

(something like that)